cfgen/cfgen

```diff
@@ -404,6 +404,9 @@ def enrich_cluster_desc(desc: Dict[str, Any], mock_p: bool) -> None:
         if 'm0_servers' not in node:
             continue
 
+        m0d = node['m0_servers']
```

Comment:
If there is no `m0_servers` key in this dict, `KeyError` will be thrown immediately (so consequent check for `is None` doesn't make sense).

https://stackoverflow.com/questions/11041405/why-dict-getkey-instead-of-dictkey

Reply:
Done!